github/orgs_credential_authorizations.go

```diff
@@ -0,0 +1,90 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+// CredentialAuthorization represents a credential authorized through SAML SSO
+type CredentialAuthorization struct {
+	// User login that owns the underlying credential.
+	Login *string `json:"login"`
+
+	// Unique identifier for the credential.
+	CredentialID *int64 `json:"credential_id"`
+
+	// Human-readable description of the credential type.
+	CredentialType *string `json:"credential_type"`
+
+	// Last eight characters of the credential.
+	// Only included in responses with credential_type of personal access token.
+	TokenLastEight *string `json:"token_last_eight"`
+
+	// Date when the credential was authorized for use.
+	CredentialAuthorizedAt *Timestamp `json:"credential_authorized_at"`
+
+	// Date when the credential was last accessed.
+	// May be null if it was never accessed.
+	CredentialAccessedAt *Timestamp `json:"credential_accessed_at"`
+
+	// List of oauth scopes the token has been granted.
+	Scopes []string `json:"scopes"`
```

Comment:
Please add `,omitempty` to all fields in this struct, since they are all reference types.

```suggestion
	Scopes []string `json:"scopes,omitempty"`
```